tests/functional/event_handler/test_openapi_validation_middleware.py

```diff
@@ -0,0 +1,167 @@
+import json
+
+from pydantic import BaseModel
+from typing_extensions import Annotated
```

Comment:
We have some types we can import directly for `typing` and we are trying to put all this import from `typing_extensions` in a single file. Please import from https://github.com/aws-powertools/powertools-lambda-python/blob/develop/aws_lambda_powertools/shared/types.py